packages/protocol/contracts/L1/verifiers/PseZkVerifier.sol

```diff
@@ -21,6 +23,19 @@ contract PseZkVerifier is EssentialContract, IVerifier {
 
     error L1_INVALID_PROOF();
 
+    struct PointProof {
+        bytes32 txListHash;
```

Comment:
Why not call this one blobHash as well?

Reply:
My understanding is that this value is the keccak hash of the RLP-encoded txList, provided by the actual zk prover. When 4844 blob is used for DA, the protocol no longer care about the keccak hash of the RLP-encoded txList, but this value is used by the protocol to calculate a random x (the instance value in our impl).

Let me know if you understand it differently.

Reply:
Ah yeah, I didn't see it was used like that! 

I think the keccak hash approach makes sense because it's easy to do and we need to support hashing the data anyway for the calldata method. There are a couple of other options but they are more complicated: https://github.com/taikoxyz/zkevm-circuits/pull/106#issuecomment-1752993436

Reply:
`the keccak hash approach makes sense` => no change? Sorry please be more specific :)

Reply:
Yep no changes!